### PR TITLE
Libra some issue

### PR DESCRIPTION
**Pull request by minhquan4080**
#64 #78 #76 #70 #71 #68 #72 #79 #75

### CODE REVIEW

**Comment by lhhieu89** [comment]
@khoainx Please help me review this pull request. Thanks!

File: app/controllers/user/ideas_controller.rb [hunk 1]
```diff
@@ -14,10 +14,23 @@ def index
...
+  def load_more
+    page = params[:page].to_i + 1
+    ideas = Idea.all
+    ideas = ideas.by_keyword(params[:q]) if params[:q].present?
+    ideas = ideas.by_category(params[:c]) if params[:c].present?
+    ideas = ideas.by_country(params[:country])if params[:country].present?
+    ideas = ideas.by_category(params[:c]) if params[:c].present?
+    @ideas = ideas.page(page).order('created_at desc')
+    render layout: false
+  end
```
Comment (lhhieu89) [inline app/controllers/user/ideas_controller.rb]: It's the same code with method `index`, please create new method to handle this one

File: app/views/api/idea/comments.html.erb [hunk 2]
```diff
@@ -20,17 +30,27 @@
         <div class="clearfix"></div>
       </div>
       <div style="margin-top:10px" id="reply_<%= comment.id %>">
-        <% comment.comments.each do |reply| %>
+        <% comment.comments.order('created_at ASC').each do |reply| %>
```
Comment (lhhieu89) [inline app/views/api/idea/comments.html.erb]: Move it to the model

File: app/views/user/ideas/_comment.html.erb [hunk 3]
```diff
@@ -18,14 +26,23 @@
       <div class="clearfix"></div>
     </div>
     <div id="reply_<%= comment.id %>">
-      <% comment.comments.each do |reply| %>
+      <% comment.comments.order('created_at DESC').each do |reply| %>
```
Comment (lhhieu89) [inline app/views/user/ideas/_comment.html.erb]: Move to the model, too

**Comment by minhquan4080** [comment]
@lhhieu89  please check it. thanks

**Comment by lhhieu89** [comment]
Ok, thanks @minhquan4080

**Pull request merged**